src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -92,13 +92,13 @@ public CursorsController(final CursorsService cursorsService,
         if (cursors.isEmpty()) {
             throw new CursorsAreEmptyException();
         }
-        final Scope commitScope = TracingService.activateSpan(request, false);
-        commitScope.span().setOperationName("commit_events");
+        final Span commitSpan = TracingService.activateSpan(request, false)
```

Comment:
could we avoid completely span activation ? I do not see the reason to do it, you could just start it in the filter